engine/src/test/scala/cromwell/engine/io/IoActorSpec.scala

```diff
@@ -20,6 +20,8 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
+class ConcreteRetry extends RetryableRequestSupport
```

Comment:
Any reason this is a trait instead of a utility `object` if it only contains static methods?